packages/jest-environment-jsdom/src/index.ts

```diff
@@ -119,7 +119,6 @@ class JSDOMEnvironment implements JestEnvironment {
       }
     }
     this.errorEventListener = null;
-    this.global = null;
```

Comment:
We set it to `null` to properly GC it. Please don't remove it :)

Reply:
We don’t do it in `jest-environment-node`, right? (https://github.com/facebook/jest/blob/31e06e8a747241b1a3a2d25dab01f4ff8faf5106/packages/jest-environment-node/src/index.ts#L87) if missing please give me to add to it!
and when environment teardown, this class will auto clean because I don’t see anywhere use it after! (https://github.com/facebook/jest/blob/1f280d89f4089770078623cd7fa51c217875de97/packages/jest-runner/src/runTest.ts#L278)

If I thinking incorrect, please give me a comment. Thank you very much!

Reply:
Tests might keep a reference to it since it's the same `global` as within the tests.

Not sure why https://github.com/facebook/jest/pull/2021 removed the cleanup from node env - possibly because the global is less leaky from an explicit sandbox than from whatever JSDOM does to get a global reference?

Reply:
Just keep the assignment to be safe? Probably better to just set all the properties to `null` than to gamble which might not need it ^^